entrypoint_scripts/common/common-os.sh

```diff
@@ -89,7 +89,7 @@ bootstrap_install() {
 	   # Install user isn't root,  check for sudo privileges
 	   echo "  Checking for sudo access, you may be prompted for your password"
        sudo -v 2>/dev/null
-       SUDO_CHECK=`sudo -v | wc -l`
+       SUDO_CHECK=`sudo -v | wc -l | awk '{$1=$1; print}'`
```

Comment:
@csansone-handy was this change intended?

Reply:
Yes it was. I was trying to run something locally (macOS Mojave) and I was having issues. Some versions of `wc -l` will emit a tab in the front so you will get `<tab>0` instead of `0`. The  `| awk '{$1=$1; print}'` just trims the whitespace

Reply:
For example:

```
$ uname -v                                                                                                                                                                                            
Darwin Kernel Version 18.7.0: Tue Aug 20 16:57:14 PDT 2019; root:xnu-4903.271.2~2/RELEASE_X86_64
$ cat ~/somefile | wc -l                                                                                                                                               
     559
```
vs.

```
$ uname -a
Linux 232842ec56f5 4.9.184-linuxkit #1 SMP Tue Jul 2 22:58:16 UTC 2019 x86_64 GNU/Linux
$ cat manage.py | wc -l
10
```